components/automate-ui/src/app/entities/users/user.reducer.ts

```diff
@@ -64,14 +78,14 @@ export function userEntityReducer(state: UserEntityState = UserEntityInitialStat
           changes: action.payload}, state));
 
     case UserActionTypes.DELETE:
-      return set('status', EntityStatus.loading, state);
+      return set('deleteStatus', EntityStatus.loading, state);
```

Comment:
I really like that you added `deleteStatus` here. 🎉

Reply:
it's been a long time coming 😅